src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_params.py

```diff
@@ -3,10 +3,26 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 #---------------------------------------------------------------------------------------------
 
+import platform
+
 from azure.cli.core.commands import register_cli_argument
 from azure.cli.core.commands.parameters import (
     name_type,
     resource_group_name_type)
+from azure.cli.core._util import CLIError
+
+def _get_default_install_location(exeName):
+    system = platform.system()
+    if system == 'Windows':
+        install_location = 'C:\\Program Files\\{}.exe'.format(exeName)
+    elif system == 'Linux' or system == 'Darwin':
+        install_location = '/usr/local/bin/{}'.format(exeName)
+    else:
+        raise CLIError('Proxy server ({}) does not exist on the cluster.'.format(system))
```

Comment:
Here, should just set `install_location=None`.
It would be too soon to raise a CLIError at this stage as it'll cause the 'acs' module to not load.

Then, in the `dcos_install_cli()` method, you'd have a check for:
```
def dcos_install_cli(install_location=None):
    if not install_location:
        raise CLIError("No install location specified and it could not be determined from the current platform '{}'".format(system))
```